hedera-node/hedera-app-spi/src/testFixtures/java/com/hedera/node/app/spi/fixtures/TestBase.java

```diff
@@ -83,6 +83,21 @@ public final Bytes randomBytes(int length) {
         return Bytes.wrap(randomBytes(rand, length));
     }
 
+    /**
+     * Generates some random bytes
+     *
+     * @param length The number of bytes to generate.
+     * @return Some random bytes.
+     */
+    @NonNull
+    public final byte[] randomByteArray(int length) {
```

Comment:
nit : should we probably use `randomBytes(rand, length)` inside this ?

Reply:
I would apply this change but if I do I lose my approvers 😡 . Next time!